src/class-wpml-gutenberg-integration.php

```diff
@@ -121,7 +121,7 @@ public function string_translated(
 			}
 
 			$this->sitepress->switch_lang( $lang );
-			wp_update_post( array( 'ID' => $translated_post_id, 'post_content' => $content ) );
+			wp_update_post( array( 'ID' => $translated_post_id, 'post_content' => addslashes( $content ) ) ;
```

Comment:
@kkarpieszuk, I am not sure this is the right way to fix what you mentioned in the description.  This change can affect all posts blocks, not only ACF. I suggest that you remove it from this PR and open a ticket in YT.